src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -142,7 +142,7 @@ private DenseMatrix initRhs(LfNetwork lfNetwork, EquationSystem equationSystem,
             for (Map.Entry<String, Number> busAndInjection : configuration.busInjectionById().entrySet()) {
                 LfBus lfBus = lfNetwork.getBusById(busAndInjection.getKey());
                 if (lfBus.isSlack()) {
-                    throw new PowsyblException("Cannot analyse sensitivity of slack bus");
```

Comment:
I am not sure to get why this line is not cover by our test.

Reply:
It is because we never reach this line, the slack bus will never be inside the businjectionbyId. But the exception here stays for the safety of future developments